op-proposer/proposer/driver.go

```diff
@@ -323,12 +384,16 @@ func (l *L2OutputSubmitter) FetchOutput(ctx context.Context, block uint64) (sour
 	return output, nil
 }
 
-// ProposeL2OutputTxData creates the transaction data for the ProposeL2Output function
+// ProposeL2OutputTxData creates proposal transaction data for L2OO.
 func (l *L2OutputSubmitter) ProposeL2OutputTxData(output source.Proposal) ([]byte, error) {
 	return proposeL2OutputTxData(l.l2ooABI, output)
 }
 
-// proposeL2OutputTxData creates the transaction data for the ProposeL2Output function
+// proposeL2OutputTxData creates proposal transaction data for L2OO.
+//
+// Errors if the ABI packing fails.
+//
+// TODO: do we need this? The `ProposeL2OutputTxData` function wraps this.
```

Comment:
nit: TODO. This function is useful to have to avoid re-parsing the embedded contract ABI. And it's also used by this [test](https://github.com/ethereum-optimism/optimism/blob/develop/op-proposer/proposer/abi_test.go#L72).